src/envo/envo-edit.owl

```diff
@@ -4115,6 +4127,23 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.ob
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001307> "Definition modified from 'surrounded by'.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001307> "partially_surrounded_by")
 
+# Object Property: <http://purl.obolibrary.org/obo/ENVO_02000139> (used for transport of)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000139> "A relation between a material entity and a material where the function of the material entity is realized as a process that results in the movement of the material.")
```

Comment:
> A relation between a material entity and a material where the function of the material entity is realized as a process that results in the movement of the material

Comments:
* I don't think functions can be realised "as" processes
* many non-utilitarian relations can result in the movement of a material

Questions:
* This appears to be shorthand to avoid creating the function class and linking to the transport process itself correct? Is there documentation on how to formulate these shorthand relations?
* Are you keeping this broad enough to allow it to be used for things like "xylem used_for_transport_of water" ? Or does this have to be intentional usage?

I'd rather have the actual functions and processes, especially if there's no convention or documentation on how these shorthand relations should be understood (is there?).